src/main/java/seedu/revised/parser/FlashcardParser.java

```diff
@@ -19,22 +19,28 @@
      */
     public static FlashcardCommand parse(String fullCommand) {
         String[] tokens = fullCommand.split(" ");
-        String command = tokens[0];
+        String fullCommandLowerCase = fullCommand.toLowerCase();
+        String command = tokens[0].toLowerCase();
 
-        if (fullCommand.equals("exit")) {
-            return new ExitFlashcardCommand();
-        } else if (fullCommand.equals("list")) {
-            return new ListFlashcardCommand();
-        } else if (fullCommand.equals("list all")) {
-            return new ListAllFlashcardCommand();
-        } else if (command.equals("add")) {
+
+        switch (command) {
+        case "add":
             return new AddFlashcardCommand(fullCommand);
-        } else if (command.equals("delete")) {
+        case "delete":
             return new DeleteFlashcardCommand(fullCommand);
-        } else if (fullCommand.equals("help")) {
+        case "help":
```

Comment:
In this way help command with any arguments will execute the help command, for example `help xxx` will also show the help list. If this is intended then just ignore this.

Reply:
working as intended